Rakefile

```diff
@@ -11,17 +11,25 @@ def brew_install(package, *args)
     # brew did not error out, verify tmux is greater than 1.8
     # e.g. brew_tmux_query = 'tmux 1.9a'
     installed_version = versions.split(/\n/).first.split(' ')[1]
-    unless version_match?(options[:version], installed_version)
+    unless version_match?(options[:requires], installed_version)
       sh "brew upgrade #{package} #{args.join ' '}"
     end
   end
 end
 
-def version_match?(requirement, version)
+def version_match?(requirement, v)
   # This is a hack, but it lets us avoid a gem dep for version checking.
+  version = parse_version(v)
```

Comment:
It's not clear to me that this additional check is necessary. I tested in irb and discovered
`Gem::Dependency.new('', '>= 1.8').match?('', '1.9a') # => true` which is what we expect.